src/platforms/hubspot/actions/mappers.ts

```diff
@@ -0,0 +1,198 @@
+import {
+  HubspotAssociationResponse,
+  HubspotCall,
+  HubspotCompany,
+  HubspotContact,
+  HubspotContactList,
+  HubspotContactListContact,
+  HubspotDeal,
+  HubspotEmail,
+  HubspotMeeting,
+  HubspotNote,
+  HubspotOwner,
+  HubspotProperty,
+  HubspotTask,
+} from '@/platforms/hubspot/schemas';
+
+export const transformOwner = (owner: HubspotOwner) => {
+  return {
+    id: owner.id,
+    createdAt: owner.createdAt,
+    updatedAt: owner.updatedAt,
+    firstName: owner.firstName,
+    lastName: owner.lastName,
+    email: owner.email,
+  };
+};
```

Comment:
is this function doing anything?

Reply:
lol it's not, but I mean I figured we should still keep it